src/modules/mount/main.py

```diff
@@ -307,6 +307,12 @@ def mount_partition(root_mount_point, partition, partitions, mount_options, moun
                                         mount_option) != 0:
                 libcalamares.utils.warning("Cannot mount {}".format(device))
 
+def enable_swap_partitions(swap_partitions):
+    try:
+        for swap_device in swap_partitions:
+            libcalamares.utils.host_env_process_output(["swapon", swap_device])
+    except subprocess.CalledProcessError:
+        libcalamares.utils.warning("Failed to swapon " + swap_device) 
```

Comment:
We generally use `.format()`, or f-strings in the 3.3 series; this is *ok*, but could be nicer.

Reply:
In case you aren't famliair with f strings, it would look like this:
```
libcalamares.utils.warning(f"Failed to swapon {swap_device}")
```